data/hai/hai reveal 7 epilogue.txt

```diff
@@ -0,0 +1,820 @@
+			sprite planet/io
+			distance 667.205
+			period 166.95
+			object Mooncreek
+				sprite "planet/station hai eight geocoris"
+				distance 212
+				period 31.1964
+		object
+			sprite planet/ice8
+			distance 1271.3
+			period 439.104
+		object
+			sprite planet/gas1
+			distance 2131.94
+			period 953.584
+			object
+				sprite planet/rock0
+				distance 244
+				period 13.4426
+		add fleet "Anti-Pirate Hai (Wormhole Travel)" 2000
+	planet Mooncreek
+		government "Hai"
+		remove attributes uninhabited
+		description `This station is made of eight Geocoris ships dismantled and reassembled into a spherical structure with external storage and landing platforms. In this dangerous place critical tactical systems have been installed first, with only minimal life support an amenities. Other than the docking and repair facilities, its main focus is a vast integrated holographic projection system which projects false security ships into space to confuse attackers and draw fire from the actual defenders of the station.`
+		security 0.4
+
+
+mission "HRF 05 Escort Republic Outfitters"
+	landing
+	name "Mooncreek Construction"
+	description "Escort outfitters to build up Mooncreek station."
+	source "Maelstrom"
+	destination "Mooncreek"
+	on visit
+		dialog `You have arrived without the convoy you are escorting. You must take off and wait for it to continue.`
+	to offer
+		has "HRF 04 Escort Republic Outfitters: done"
+	on offer
+		conversation
+			`When you arrive on Maelstrom you are greeted by a representative of Lovelace Labs. They and their Navy escort are ready to depart on their PR flight immediately.`
+				accept
+	npc accompany save
+		personality escort heroic
+		government Republic
+		ship "Behemoth (Heavy)" "R.N.S. Bungles"
+		ship "Behemoth (Heavy)" "R.N.S. Humble"
+		ship "Behemoth (Heavy)" "R.N.S. Sensible"
+		ship "Behemoth (Heavy)" "R.N.S. Temperate"
+	npc accompany save
+		personality escort heroic
+		government Republic
+		fleet
+			names "republic capital"
+			fighters "republic fighter"
+			variant
+				"Carrier"
+				"Lance" 4
+				"Combat Drone" 6
+				"Frigate" 2
+				"Gunboat" 3
+				"Rainmaker" 3
+
+
+mission "HRF 06 Escort Syndicate Outfitters"
+	landing
+	name "Mooncreek Construction"
+	description "Escort outfitters to build up Mooncreek station."
+	source "Mooncreek"
+	destination "Sunracer"
+	to offer
+		has "HRF 05 Escort Republic Outfitters: done"
+	on offer
+		event "mooncreek republic garrison"
+		conversation
+			`The Republic garrison and Lovelace transports arrive to suitable fanfare and media pomp. Xilin and Hiyamaharu appear to have hitched a ride to Mooncreek in your abence and they greet you in what passes for a spaceport that is not yet serving drinks.`
+			`	"Good job there <last>," says Xilin.`
+			`	"I know it's trivial but don't underestimate the impact of a stately arrival. After the crisis last time it's really very important to penetrate the public consciousness and get people to know what's going on."`
```

Comment:
```suggestion
			`The Republic garrison and Lovelace transports arrive to suitable fanfare and media pomp. Xilin and Hiyamaharu appear to have hitched a ride to Mooncreek in your absence, and they greet you in what passes for a spaceport that is not yet serving drinks.`
			`	"Good job out there, <last>," says Xilin.`
			`	"I know it's trivial, but don't underestimate the impact of a stately arrival. After the crisis last time, it's really very important to penetrate the public consciousness and get people aware of what's going on."`
```

Reply:
Maybe "to appreciate" is more appropriate. I tend to use "to know" more in the biblical sense of "to understand" than in the more vague colloquial sense.

Reply:
Gotcha. I think this resulted from me misunderstanding what's being referenced as `what's going on`. Do you mind adding this bit of clarification too?
```suggestion
			`The Republic garrison and Lovelace transports arrive to suitable fanfare and media pomp. Xilin and Hiyamaharu appear to have hitched a ride to Mooncreek in your absence, and they greet you in what passes for a spaceport that is not yet serving drinks.`
			`	"Good job out there, <last>," says Xilin.`
			`	"I know it's trivial, but don't underestimate the impact of a stately arrival. After the crisis last time, it's really very important to penetrate the public consciousness and get people to appreciate the magnitude of these steps."`
```